include/deal.II/base/geometry_info.h

```diff
@@ -2312,6 +2312,19 @@ struct GeometryInfo
                              const bool         face_flip        = false,
                              const bool         face_rotation    = false);
 
+  static unsigned int
+  standard_to_real_line_vertex(const unsigned int line,
+                               const bool         face_orientation = true);
+
+  static std::array<unsigned int, 2>
+  standard_corner_to_line_vertex_index(const unsigned int corner);
+
+  static std::array<unsigned int, 2>
+  standard_corner_to_quad_vertex_index(const unsigned int corner);
```

Comment:
We don't use the term "corner" anywhere in deal.II. Since you've got no documentation, I can't figure out what it's supposed to mean here, but why not use "vertex"?

Reply:
> We don't use the term "corner" anywhere in deal.II

That is not completely true (here vertex and corner are used as synonym):

https://github.com/dealii/dealii/blob/159f21c4d12201ffa9ac190202b0ba0997a003b5/include/deal.II/grid/tria_accessor.templates.h#L1089-L1145

I'll add some documentation later.